### ISSUE TITLE
Session metadata query error in sqlsrv

### ISSUE THREAD

**Issue by HLeithner** [joomla/joomla-cms#25536]
The metadata insert query seam to fail on sqlsrv.

```SQL
INSERT INTO [#__session] ([session_id],[guest],[time],[userid],[username],[client_id])VALUES (N'qc4jsu8lppn2672m2rb5hadjg8', 1, 1562944378, 0, N'', 0)
```

Error message:
JDatabaseExceptionExecuting: [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Implicit conversion from data type nvarchar to varbinary is not allowed. Use the CONVERT function to run this query.

### Steps to reproduce the issue
Install Joomla 3.9.10 on Mssql server 2017 and open the frontpage. In 1 of 3 call the request failed with "Error: Failed to start application: Error starting the session.".

### Expected result
Nice page

### Actual result
Error: Failed to start application: Error starting the session.

### System information (as much as possible)
MSSQL Server 2017 Linux

**Comment by SharkyKZ** [comment]
There's this closed PR #23213.

**Comment by HLeithner** [comment]
Solved by PR #23213
